src/components/ADempiere/Form/ProductInfo/productList.vue

```diff
@@ -109,7 +160,12 @@
         width="300"
       >
         <template slot-scope="scope">
-          {{ formatPrice(getTaxAmount(scope.row.priceStandard, scope.row.taxRate.rate) + scope.row.priceStandard, scope.row.currency.iSOCode) }}
+          {{
```

Comment:
Just can see change for soiurce code with the same functionality, please let it without changes